Pop_Quiz.txt

```diff
@@ -15,6 +15,7 @@ Pop Quiz
 7. What is the difference between git add . and git commit -a?
 
 8. What does rm do? How is it used with directories?
+Remove files from the directory. On cmd line type rm directoryName/
```

Comment:
`rm -r directoryName/` or `rm file`

`rm directoryName/` will result in an error

Reply:
rm -r directoryName/ or rm file